src/backend/expungeservice/models/charge_types/person_felony.py

```diff
@@ -0,0 +1,177 @@
+        "166070",  # Aggravated Harassment;
+        "166087",  # Abuse of Corpse I;
+        # "166165",  # Bias Crime I;
+        # "166220",  # Unlawful Use of a Weapon;
+        "166275",  # Inmate In Possession of Weapon;
+        # "1663853", # [contains subsection] # Felony Possession of a Hoax Destructive Device;
+        "166643",  # Unlawful Possession of Soft Body Armor as defined in Crime Category 6 (OAR 213-018-0090(1));
+        "167012",  # Promoting Prostitution;
+        "167017",  # Compelling Prostitution;
+        "167057",  # Luring a Minor;
+        # "1673204", # [contains subsection] # Felony Animal Abuse I;
+        "167322",  # Aggravated Animal Abuse I;
+        "468951",  # Environmental Endangerment;
+        # "4757526A", # [contains subsection] # Manufacturing or Delivering a Schedule IV Controlled Substance Thereby Causing Death to a Person;
+        "475908",  # Causing Another to Ingest a Controlled Substance as defined in Crime Categories 8 and 9 (OAR 213-019-0007 and 0008);
+        "475910",  # Unlawful Administration of a Controlled Substance as defined in Crime Categories 5, 8, and 9 (OAR 213-019-0007, -0008, and -0011);
+        # "475B359",  # Arson Incident to Manufacture of Cannabinoid Extract I;
+        # "475B367",  # Causing Another Person to Ingest Marijuana;
+        # "475B371",  # Administration to Another Person Under 18 Years of Age;
+        # "6099903B", # [contains subsection] # Maintaining Dangerous Dog;
+        # "811705",  # Hit and Run Vehicle (Injury);
+        # "8130105", # Felony Driving Under the Influence of Intoxicants (as provided in OAR 213-004-0009);
+        # "8304752", # [contains subsection] # Hit and Run Boat;
+        # "8373652B", # [contains subsection] # Unlawful Operation of Weaponized Unmanned Aircraft System;
+        # "8373652C", # [contains subsection] # Unlawful Operation of Weaponized Unmanned Aircraft System;
+    ]
+
+    def _type_eligibility(self):
+        if self.dismissed():
+            return TypeEligibility(EligibilityStatus.ELIGIBLE, reason="Dismissals are eligible under 137.225(1)(b)")
+        elif self.convicted():
+            if self.statute in [full_statute[:6] for full_statute in self.statutes_with_subsection]:
+                return TypeEligibility(
+                    EligibilityStatus.NEEDS_MORE_ANALYSIS,
+                    reason="OECI may be missing a statute subsection which would make this charge a person crime, and thus ineligible under 137.225(5)(a)",
+                )
+            else:
+                return TypeEligibility(EligibilityStatus.INELIGIBLE, reason="Ineligible under 137.225(5)(a)")
+
+
+"""The following sections are defined in (ORS)[https://www.oregonlaws.org/] as class B felonies. Some of these have only subsections which are a class B felony.
+This list is not used directly to determine expungement eligibility because we instead use the charge level provided in the OECI record.
```

Comment:
What is the purpose of this comment?

Reply:
The implementation is in some sense overkill because the complete list of person felonies includes things that can never be B felonies. But when looking at the level of a charge we consider what is in OECI and not what is the charge level named in the statute.

Arguably, you could shorten the list to only the person felonies that could also be B felonies, which would instead by this list.. I put it here as a comment is basically as extra background info. It's here for lack of a better place to put it, since we're basically trying to keep all of the legal verbiage within the source code and then extract for user-facing docs (or in this case not, because it's kind of excessive information. )

Reply:
What is the purpose of the long list of crimes that follow then? Is that not duplicated above in the code mostly?

Reply:
It's just the actual set of statutes typically considered class B person crimes. It seemed useful to me to keep the actual list of 30 statutes that typically apply , out of the 90 defined person felonies. The code above doesn't explicitly identify the class B felonies.

Reply:
Ok